src/ontology/components/mondo-tags.owl

```diff
@@ -15528,7 +15528,6 @@
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/MONDO_0006042">
         <mondo:excluded_from_qc_check rdf:resource="http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql"/>
         <terms:conformsTo rdf:resource="http://purl.obolibrary.org/obo/mondo/patterns/specific_infectious_disease_by_location.yaml"/>
-        <terms:conformsTo rdf:resource="http://purl.obolibrary.org/obo/mondo/patterns/specific_inflammatory_disease_by_site.yaml"/>
```

Comment:
@sabrinatoro you know what this means, right? Is this okay? If yes, please feel free to merge